code/controllers/subsystem/airmachines.dm

```diff
@@ -25,13 +25,13 @@ SUBSYSTEM_DEF(airmachines)
 
 /datum/controller/subsystem/airmachines/Initialize(timeofday)
 	var/starttime = REALTIMEOFDAY
-	to_chat(world, span_debug("Airmachines: Setting up atmospheric machinery..."))
+	to_chat(world, span_debug_info("Airmachines: Setting up atmospheric machinery..."))
 	setup_atmos_machinery()
-	to_chat(world, span_debug("Airmachines: Airmachine setup completed in [(REALTIMEOFDAY- starttime) / 10] seconds!"))
+	to_chat(world, span_debug_info("Airmachines: Airmachine setup completed in [(REALTIMEOFDAY- starttime) / 10] seconds!"))
 	starttime = REALTIMEOFDAY
-	to_chat(world, span_debug("Airmachines: Creating pipenets..."))
+	to_chat(world, span_debug_info("Airmachines: Creating pipenets..."))
 	setup_pipenets()
-	to_chat(world, span_debug("Airmachines: Pipenet creation completed in [(REALTIMEOFDAY- starttime) / 10] seconds!"))
+	to_chat(world, span_debug_info("Airmachines: Pipenet creation completed in [(REALTIMEOFDAY- starttime) / 10] seconds!"))
```

Comment:
Don't make these debug spans, I like the way it looks as-is.

Reply:
Unavoidable without manually styling, as this used the old debug span.